### DISCUSSION TITLE
Mehr Import Möglichkeiten.

### DISCUSSION THREAD

**Discussion by jeGit** [bassmaster187/TeslaLogger#1133] [Ideas]
Ich würde mir mehr Möglichkeiten zum Datenimport aus anderen Lösungen wünschen.
Konkret aus TezLab. Dort bietet die ProVersion einen csv Export an.
Oder eine Beschreibung, wie csv Daten in Teslalogger importiert werden könnten.
Danke.

**Comment by frommg** [comment]
Bitte auch für Tessie, meine ganzen historischen Daten sind dort drinn. Es kann ein CSV export erstellt werden in Tessie.
